pennylane/transforms/batch_partial.py

```diff
@@ -0,0 +1,180 @@
+# Copyright 2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Contains the batch dimension transform.
```

Comment:
With having `batch_params` and `batch_inputs` too, it could be worth further mentioning that this is specific to the partial use of QNodes.

Reply:
Updated docstring to mention the partial use of QNodes.